docs/source/reference/visualization/matplotlib.rst

```diff
@@ -18,4 +18,5 @@ optuna.visualization.matplotlib
     optuna.visualization.matplotlib.plot_param_importances
     optuna.visualization.matplotlib.plot_pareto_front
     optuna.visualization.matplotlib.plot_slice
+    optuna.visualization.matplotlib.plot_timeline
```

Comment:
[Notes] The reference page was rendered as expected:
![image](https://user-images.githubusercontent.com/3255979/232654112-ce23fd0e-6776-484b-8510-329d2faa463a.png)